addons/tmf_loadouts/loadouts/US_USMC_1985.hpp

```diff
@@ -0,0 +1,493 @@
+	backPack[] = {
+		"usm_pack_762x51_bandoliers",
+		"usm_pack_762x51_ammobelts"
+	};
+	linkedItems[] += {"Binocular"};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_M2_Gun_Bag"};
+};
+
+class hmgac : car {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_M2_MiniTripod_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	backPack[] = {"CUP_B_M2_Tripod_Bag"};
+	linkedItems[] += {"Binocular"};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"usm_pack_alice"};
+	secondaryWeapon[] = {"CUP_launch_M47"};
+	magazines[] += {
+		LIST_2("CUP_Dragon_EP1_M")
+	};
+};
+
+class matac : car {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_2("CUP_Dragon_EP1_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	backpack[] = {"usm_pack_alice"};
+	linkedItems[] += {
+		"ACE_Vector",
+		"ItemGPS"
+	};
+	backpackItems[] = {
+		LIST_2("CUP_Dragon_EP1_M"),
+		"ACE_MapTools"
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Tow_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	backPack[] = {"CUP_B_TOW2_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"Binocular"};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { "MTR_GEAR" };
+	backPack[] = {"usm_pack_alice"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+
+	linkedItems[] += {
+		"Binocular",
+		"ItemGPS"
+	};
+};
+
+class samg : car {
+	displayName = "AA Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+};
+
+class samag : car {
+	displayName = "AA Assistant Missile Specialist";
+	secondaryWeapon[] = {"CUP_launch_FIM92Stinger"};
+	linkedItems[] = {"Binocular"};
+};
+
+class vg : smg {
+	displayName = "Vehicle Gunner";
+	uniform[] = {"SP_0000_Standard_Coverall_Tan"};
+	vest[] = {"CUP_V_B_Eagle_SPC_Crew"};
+	backpack[] = {};
+	headgear[] = {"SP_CVCHelmet_Tan"};
+	magazines[] = {
+		LIST_3("CUP_30Rnd_9x19_MP5"),
+		"SmokeShellPurple"
+	};
+};
+
+class sn : r {
+	displayName = "Sniper";
+	headgear[] = {"CUP_H_US_patrol_cap_desert"};
+	goggles[] = {"default"};
+	vest[] = {"CFP_Alice_Vest2"};
+	primaryWeapon[] = {"CUP_srifle_M40A3_bipod"};
```

Comment:
Only use preset rifles for backpacks and inventories. In all other cases, use scope[], bipod[] and attachment[]